src/main/java/com/zebrunner/carina/webdriver/DriverHelper.java

```diff
@@ -496,6 +506,192 @@ public void addCookie(Cookie cookie) {
         });
     }
 
+    /**
+     * add a cookie object into the driver
+     *
+     * @param name name of the cookie
+     * @param value the value of the cookie
+     * @param domain the domain of the cookie
+     * @param path the path of the cookie
+     * @param expiry when the cookie expires
+     */
+    public void addCookie(String name, String value, String domain, String path, Date expiry) {
+        addCookie(new Cookie(name, value, domain, path, expiry));
+    }
+
+    /**
+     * add a cookie object into the driver
+     *
+     * @param name name of the cookie
+     * @param value the value of the cookie
+     * @param domain the domain of the cookie
+     * @param path the path of the cookie
+     * @param expiry when the cookie expires
+     * @param isSecure if the cookie is secure or not
+     * @param isHttpOnly if it is on http or https
+     */
+    public void addCookie(String name, String value, String domain, String path, Date expiry, boolean isSecure, boolean isHttpOnly) {
```

Comment:
It is not necessary to have such method because it has a lot of parameters.
We already have addCookie method that accept org.openqa.selenium.Cookie as parameter. If user want to add cookie it can create Cookie object by himself.

Reply:
Makes sense. Removing this method.